spec/student_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe Student do
+  before :each do
+    @student = Student.new 23, 'Muhammad Asif', true
+  end
```

Comment:
- [ ] In addition to all the written tests, can you please write tests for this missing method (`play_hooky`)